.github/workflows/build_branch.yaml

```diff
@@ -8,6 +8,9 @@ on:
 
 jobs:
   build-and-push-to-npm:
+    permissions:
+      contents: "read"
+      packages: "write"
     name: Bygg og push
```

Comment:
Skal det være fnutter rundt read og write?

https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

Reply:
Ser ikke sånn ut, nei